engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableBlending.java

```diff
@@ -26,6 +26,7 @@
  */
 public class EnableBlending extends SetStateParameter {
     private static final int PARAMETER = GL_BLEND;
+    private static final String PARAMETER_NAME = "GL_BLEND";
```

Comment:
I wonder what the utility of these fields are versus simply having:

`disablingTask = new DisableStateParameterTask(GL_BLEND, "GL_BLEND");`

but I'm ok with leaving them as they are to get through this stage of the PR and progress to the next stage.